### ISSUE TITLE
Remove spring libs-release repo

### ISSUE THREAD

**Issue by aartoni** [icescrum/iceScrum#49]
Upon further investigation on pull request #45, I noticed that the Spring `libs-release` repository, is being closed to the public (see [this announcement](https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020)).

This means that the current configuration won't be able to resolve some of the dependencies for the `iceScrum` and `iceScrum-core` projects, namely `http-builder`.

Note that I did some testing and noticed that neither downgrading `http-builder` to 0.7.1, nor changing `libs-release` to `release` fixed the problem.

Actually the downgrade allowed me to run a successful `grails compile`, too bad `grails war` failed anyway complaining that `http-builder` 0.7.2 cannot be found (weird since I already changed it in the `BuildConfig.groovy` file of the `iceScrum-core` project).

**Comment by noullet** [comment]
@aartoni Thanks for the information regarding spring repositories! It seems that I tried it and answered in the pull request just before they added the authentication (on 12 November).

Anyway since http-builder 0.7.2 is not in maven central, we added the jcenter respository which has it. It works fine for us, can you try again?

**Comment by noullet** [comment]
Hi, any news on your side?

**Comment by aartoni** [comment]
Hi @noullet, sorry if I answer you only now, I thank you infinitely for your availability. I managed to get a .war that I haven't had the chance to test yet, I'll try it in a few months when I'm less busy.

In the meantime I take this opportunity to ask you how you manage to produce the fat jar instead of the .war, currently I am using the `grails war` command. If I'm not mistaken iceScrum uses Grails 2.x, while the `grails assemble` compilation task was added in Grails 3.
